packages/xo-server/src/_promiseAll.js

```diff
@@ -0,0 +1,20 @@
+// this function is like `Promise.all`, the only difference is that it not
+// short-circuits when a promise rejects
+export const all = async iterable => {
```

Comment:
I think the name is not clear enough, maybe `waitAll`?

And mutualize with the code in `backups-ng/index.js` 😉